schema/deploy/computed_columns/project_revision_anticipated_funding_amount_by_fiscal_year.sql

```diff
@@ -0,0 +1,42 @@
+-- Deploy cif:computed_columns/project_revision_anticipated_funding_amount_by_fiscal_year to pg
+
+begin;
+
+create or replace function cif.project_revision_anticipated_funding_amount_per_fiscal_year(project_revision cif.project_revision)
+returns setof cif.sum_by_fiscal_year
+as
+$computed_column$
+
+with form_changes as (
+    select * from cif.form_change
+    where project_revision_id = $1.id
+    and form_data_schema_name='cif'
+    and form_data_table_name='reporting_requirement'
+    and (new_form_data->>'reportType') in (select name from cif.report_type where has_expenses=true)
```

Comment:
@dleard, it does only need to be milestones in these calculations (TEIMP payment is holdbacks, so those were already paid to csnr). Would you suggest changing this function then?